mysql/create-aion.sql

```diff
@@ -358,6 +358,14 @@ create table account_stats
 	primary key (block_number, address)
 );
 
+CREATE TABLE `circulating_supply` (
+  `timestamp` datetime NOT NULL,
+  `supply` decimal(16,1) DEFAULT NULL,
```

Comment:
Any idea why we choose `decimal(16,1)` here? Would it not be more reasonable to store as much precision as possible and truncate in code.

Reply:
Also I feel this field should be `NOT NULL`, since it's very meaningful information for the table

Reply:
I just keet the current structure, with the intention of make big changes, but I will fix and add the null